Source/WebCore/platform/network/ResourceRequestBase.h

```diff
@@ -264,6 +264,8 @@ class ResourceRequestBase {
     bool useNetworkConnectionIntegrity() const { return m_requestData.m_useNetworkConnectionIntegrity; }
     WEBCORE_EXPORT void setUseNetworkConnectionIntegrity(bool);
 
+    bool urlContainsFragmentIdentifier() const { return m_requestData.m_firstPartyForCookies.hasFragmentIdentifier(); }
```

Comment:
using `m_firstPartyForCookies` isn't right here. Call sites should use `request.url().hasFragmentIdentifier()`.

Reply:
Removed this. Thanks was not sure how to access it otherwise.